hudi-utilities/src/test/java/org/apache/hudi/utilities/TestHoodieIndexer.java

```diff
@@ -75,6 +76,7 @@
 import static org.junit.jupiter.api.Assertions.assertFalse;
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
+@Disabled
```

Comment:
If this is due to multiple Spark context exception, we should use `SparkClientFunctionalTestHarness` to rewrite this test and avoid initializing the spark context again, to fix the tests.